themes/drivers/pagenav.html

```diff
@@ -3,9 +3,3 @@
 </h3>
 
 {{ toctree( titles_only=1) }}
-
-<div class="nav-footer">
```

Comment:
Thanks for removing this.  Every time I saw it, I kept saying "We should remove this."